### PR TITLE
[BPK-1244] Add support for Backpack text styles

### PR DESCRIPTION
**Pull request by k0nserv**
Adds the Backpack text styles

### CODE REVIEW

**Comment by k0nserv** [comment]
Any idea about this https://travis-ci.org/Skyscanner/backpack-android/builds/338560287?utm_source=github_status&utm_medium=notification @matteo-hertel? Per the docs `getComplexUnit` should be available since API level 22 and we seem to be using API level 27 on Travis

File: .idea/modules.xml [hunk 1]
```diff
@@ -3,8 +3,11 @@
   <component name="ProjectModuleManager">
     <modules>
       <module fileurl="file://$PROJECT_DIR$/Backpack/Backpack.iml" filepath="$PROJECT_DIR$/Backpack/Backpack.iml" />
+      <module fileurl="file://$PROJECT_DIR$/Backpack/Backpack.iml" filepath="$PROJECT_DIR$/Backpack/Backpack.iml" />
+      <module fileurl="file://$PROJECT_DIR$/app/app.iml" filepath="$PROJECT_DIR$/app/app.iml" />
```
Comment (matteo-hertel) [inline .idea/modules.xml]: these two are still a mistery to me, are the exact same as the others, why do they get added?

File: app/src/androidTest/java/net/skyscanner/backpack/BpkResources.kt [hunk 2]
```diff
@@ -3,6 +3,8 @@ package net.skyscanner.backpack
 import android.content.res.Resources
 import android.support.test.InstrumentationRegistry
 import android.support.test.runner.AndroidJUnit4
+import android.util.Log
```
Comment (matteo-hertel) [inline app/src/androidTest/java/net/skyscanner/backpack/BpkResources.kt]: this is probaby not needed anymore

File: package.json [hunk 3]
```diff
@@ -44,6 +44,7 @@
     "eslint-plugin-jsx-a11y": "^6.0.3",
     "eslint-plugin-prettier": "^2.6.0",
     "eslint-plugin-react": "^7.6.1",
+    "eslint_d": "^5.2.2",
```
Comment (matteo-hertel) [inline package.json]: what this one does?

File: .idea/modules.xml (hunk 1 above)
Comment (k0nserv) [reply .idea/modules.xml]: I have no idea

File: app/src/androidTest/java/net/skyscanner/backpack/BpkResources.kt (hunk 2 above)
Comment (k0nserv) [reply app/src/androidTest/java/net/skyscanner/backpack/BpkResources.kt]: Yup

File: package.json (hunk 3 above)
Comment (k0nserv) [reply package.json]: Make eslint work with my setup and ensures we all use the same version of eslint_d

**Pull request merged**